files/js-cookie/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "js-cookie",
+  "repo": "js-cookie/js-cookie",
+  "files": {
+    "include": ["src/js.cookie.js"]
```

Comment:
Is there any chance you could add the min file somewhere? The CDN is all about the speed, but serving an unminified file defeats that purpose.